### DISCUSSION TITLE
Can i use it for my startup

### DISCUSSION THREAD

**Discussion by mageshyt** [pocketbase/pocketbase#2528] [Q&A]
Hello, this is Magesh. I am working on a project where the owner can upload their rooms, and users can book them, similar to platforms like Oyo or Airbnb. Currently, I am using MongoDB to store data and Firebase to store images. I switching to PocketBase will be good options or i should wait untill the launch of  stable version .

**Comment by ganigeorgiev** [comment]
It is difficult to answer because it depends on a lot of factors - the amount of traffic you expect (_testbed and benchmarks will be available soon_), how big is your database, whether you are fine with using Go in case something custom need to be implemented (_js app hooks are planned for the next release_) OR using a SSR/node framework as a middleware like SvelteKit, Nuxt, Next.js, etc.

Why do you want to migrate from your current stack in the first place? Did you hit some limitation?

If your app is mission-critical and/or you are not OK with manually applying some migration steps from time to time, then it may be better to wait until we reach a little more stable version.

**Reply by mageshyt** [reply]
We are using Next.js for the frontend, and our app is still under development. Initially, we plan to launch it for a small group of users and gradually expand its reach. Internationalization is not a priority at the moment. Since our app is not yet in production, there is no need to worry about migrations. I really love pockBase simplicity and usage and Api docs .
